.github/workflows/operate-update-docs-screenshots.yml

```diff
@@ -1,5 +1,38 @@
 name: Operate Update screenshots
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "stable/**"
+      - "release/**"
```

Comment:
I think it's enough that the workflow only runs on `main` branch, because we would never use screenshots from stable or release branches.